Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG GO_VERSION="1.19"
 #--------Build KSOPS and Kustomize-----------#
 #--------------------------------------------#
 
-FROM golang:$GO_VERSION
+# Stage 1: Build KSOPS and Kustomize
+FROM golang:$GO_VERSION AS builder
 
 LABEL org.opencontainers.image.source="https://github.com/viaduct-ai/kustomize-sops"
```

Comment:
The `LABEL` needs to be moved down to the final image

Reply:
https://github.com/viaduct-ai/kustomize-sops/pull/181